### PR TITLE
PLT-5448 - Updating styling of server tabs

### PR DESCRIPTION
**Pull request by asaadmahmood**
**Summary**
PLT-5448 - Updating styling of server tabs

**Issue link**
https://mattermost.atlassian.net/browse/PLT-5448

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @asaadmahmood! Looks like the mention badge flows over the tab border.

image

**Comment by jasonblais** [comment]
@jnugh How much work would it be to add a tooltip for the plus-icon? We discussed it before, but you mentioned you'd need help with the styling first

**Comment by jnugh** [comment]
What kind of tooltip do you mean? Like this https://react-bootstrap.github.io/components.html#tooltips or these standard browser hover tooltips

**Comment by jasonblais** [comment]
Something like the browser hover tooltips

image

It can probably match the design when hovering over the minimize/restore/close buttons

image

File: src/browser/components/TabBar.jsx [hunk 1]
```diff
@@ -81,10 +81,24 @@ class TabBar extends React.Component {
...
+      borderRadius: '2px 2px 0 0',
+      outline: 'none'
+    };
+
     return (
       <Button
         id='tabBarAddNewTeam'
         onClick={this.props.onAddServer}
+        style={tabButton}
         bsStyle='tabButton'
```
Comment (jnugh) [inline src/browser/components/TabBar.jsx]: @jasonblais To create such a Tooltip, we can jsut add a title property here e.g.:
```javascript
title='Add a new server'
```

**Comment by asaadmahmood** [comment]
Done.

**Comment by yuya-oc** [comment]
Thanks! It looks good.

Windows:
badge_and_tooltip

Mac:
tooltip_mac

**Comment by jasonblais** [comment]
Looks good!  Only proposal is to change the tooltip text to `Add new server` so it's consistent with the Settings page.

image

**Comment by jasonblais** [comment]
Thanks @asaadmahmood ! Looks good to me

**Pull request merged**